vortex-array/src/compute/mod.rs

```diff
@@ -67,6 +69,13 @@ pub trait ComputeVTable {
         None
     }
 
+    /// Invert a boolean array.
```

Comment:
nit: prev doc string was better in that it was explicit. Like, obviously the invert fn inverts, but what does "invert" mean? `true -> false`, `false -> true`, `null -> null` is a nice explicit contract